docs/data-sources/cae.environments.md

```diff
@@ -0,0 +1,54 @@
+---
+subcategory: "Direct Connect (CAE)"
+---
+
+# huaweicloud_cae_environments
+
+Use this data source to get the list of CAE environments.
+
+## Example Usage
+
+```hcl
+data "huaweicloud_cae_environments" "test" {
+  name = "test"
+}
+```
+
+## Argument Reference
+
+The following arguments are supported:
+
+* `region` - (Optional, String) Specifies the region in which to query the data source.
+  If omitted, the provider-level region will be used.
+
+* `environment_id` - (Optional, String) Specifies the ID of the environment.
+
+* `name` - (Optional, String) Specifies the name of the environment.
+
+* `status` - (Optional, String) Specifies the status of the environment.
+
+* `enterprise_project_id` - (Optional, String) Indicates the ID of the enterprise project that the environment belongs to.
+
+## Attribute Reference
+
+In addition to all arguments above, the following attributes are exported:
+
+* `id` - The data source ID.
+
+* `environment_items` - Indicates the environments list.
+  The [environment_items](#CAE_environment_items) structure is documented below.
+
+<a name="CAE_environment_items"></a>
+The `environment_items` block supports:
+
+* `id` - The ID of the environment item.
+
+* `name` - The name of the environment item.
+
+* `status` - The status of the environment item, enum: `creating`、`finish`、`deleting`、`freeze`、`police_freeze`、`delete_failed`.
```

Comment:
```suggestion
* `status` - The status of the environment.
```
please take these values into the description of the argument `status`, and do not contain the Chinese characters.
```
The valid values are **finish**, **freeze**, **police_freeze**.
```

Reply:
done

Reply:
check your description again!

Reply:
done